core/controllers/admin.py

```diff
@@ -127,57 +132,131 @@ def get(self):
+                    if name == 'moderator_usernames':
+                        check_and_update_config_role(
+                            config_properties[name]['value'], value,
+                            feconf.ROLE_MODERATOR)
+                self.render_json({})
             elif self.payload.get('action') == 'revert_config_property':
                 config_property_id = self.payload.get('config_property_id')
                 logging.info('[ADMIN] %s reverted config property: %s' %
                              (self.user_id, config_property_id))
                 config_services.revert_property(
                     self.user_id, config_property_id)
+                self.render_json({})
             elif self.payload.get('action') == 'start_new_job':
                 for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
                     if klass.__name__ == self.payload.get('job_type'):
                         klass.enqueue(klass.create_new())
                         break
+                self.render_json({})
             elif self.payload.get('action') == 'cancel_job':
                 job_id = self.payload.get('job_id')
                 job_type = self.payload.get('job_type')
                 for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
                     if klass.__name__ == job_type:
                         klass.cancel(job_id, self.user_id)
                         break
+                self.render_json({})
             elif self.payload.get('action') == 'start_computation':
                 computation_type = self.payload.get('computation_type')
                 for klass in jobs_registry.ALL_CONTINUOUS_COMPUTATION_MANAGERS:
                     if klass.__name__ == computation_type:
                         klass.start_computation()
                         break
+                self.render_json({})
             elif self.payload.get('action') == 'stop_computation':
                 computation_type = self.payload.get('computation_type')
                 for klass in jobs_registry.ALL_CONTINUOUS_COMPUTATION_MANAGERS:
                     if klass.__name__ == computation_type:
                         klass.stop_computation(self.user_id)
                         break
+                self.render_json({})
             elif self.payload.get('action') == 'upload_topic_similarities':
                 data = self.payload.get('data')
                 recommendations_services.update_topic_similarities(data)
+                self.render_json({})
+            elif self.payload.get('action') == 'view_by_role':
+                role = self.payload.get('role')
+                users_by_role = user_services.get_usernames_by_role(role)
+                self.render_json(users_by_role)
+            elif self.payload.get('action') == 'view_role_by_username':
+                user_name = self.payload.get('username')
+                user_id = user_services.get_user_id_from_username(user_name)
+                result = {}
```

Comment:
Using user_role_dict because: I have used user_name and user_id for user's name and id, so user_role or role_of_user might be indicative to simply role(str) of the user but it is a dict instead.